neqo-transport/tests/server.rs

```diff
@@ -284,7 +284,16 @@ fn get_ticket(server: &mut Server) -> ResumptionToken {
 
     // Calling active_connections clears the set of active connections.
     assert_eq!(server.active_connections().len(), 1);
-    client.resumption_token().unwrap()
+    client
+        .events()
+        .fold(None, |res, e| {
```

Comment:
Using `fold` here rather than find_map() is a little non-intuitive.  Is there any reason why you need the last rather than the first?

Reply:
Sorry, this was needed in the previous patch (in my old draft version I needed to get the last ticket) and I just copied it.